CHANGELOG.md

```diff
@@ -0,0 +1,5148 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [5.0.0.rc1](https://github.com/openvoxproject/openbolt/tree/5.0.0.rc1) (2025-07-11)
+
+[Full Changelog](https://github.com/openvoxproject/openbolt/compare/4.0.0...5.0.0.rc1)
+
+**Breaking changes:**
+
+- Rename gem to openbolt [\#11](https://github.com/OpenVoxProject/openbolt/pull/11) ([jessereynolds](https://github.com/jessereynolds))
+- Bump minimal required Ruby from 2.5-\>3.2 [\#6](https://github.com/OpenVoxProject/openbolt/pull/6) ([bastelfreak](https://github.com/bastelfreak))
+
+**Closed issues:**
+
+- A new name for bolt! - OpenBolt [\#5](https://github.com/OpenVoxProject/openbolt/issues/5)
```

Comment:
I added skip-changelog here because we already have https://github.com/OpenVoxProject/openbolt/pull/11 in the CHANGELOG.md